src/Lachain.Storage/Repositories/SnapshotIndexRepository.cs

```diff
@@ -18,6 +22,10 @@ public SnapshotIndexRepository(IRocksDbContext dbContext, IStorageManager storag
         {
             _dbContext = dbContext;
             _storageManager = storageManager;
+            var prefix = EntryPrefix.DeleteOldSnapshotStatus.BuildPrefix();
```

Comment:
Can we add a Get Method to retrive this value ?

Reply:
Done.